src/data.js

```diff
@@ -672,6 +672,26 @@ let WorkerType = Entity.configure({
     // the time at which this worker-type should no longer be displayed
     expires:          Entity.types.Date,
   },
+}).configure({
+  version:            2,
+  properties: {
+    provisionerId:    Entity.types.String,
+    workerType:       Entity.types.String,
+    // the time at which this worker-type should no longer be displayed
+    expires:          Entity.types.Date,
+    description:      Entity.types.String,
+    stability:        Entity.types.String,
+    documentation:    Entity.types.String,
+    payloadSchema:    Entity.types.JSON,
+  },
+  migrate(item) {
+    item.description = '';
+    item.stability = 'experimental';
+    item.documentation = '';
+    item.payloadSchema = null;
```

Comment:
Do I also need to change [L97](https://github.com/helfi92/taskcluster-queue/blob/2e915b7597dd88454b1c0c1e40604219c856bdb3/src/workerinfo.js#L97) to true? I'm only asking because when I do that, the tests are failing most probably because the schema was expecting an object/null but received a boolean value.